libs/net/dal/Services/AVOverviewInstanceService.cs

```diff
@@ -29,7 +29,7 @@ public class AVOverviewInstanceService : BaseService<AVOverviewInstance, int>, I
             .Include(i => i.Template)
             .Include(i => i.Sections).ThenInclude(s => s.Source)
             .Include(i => i.Sections).ThenInclude(s => s.Series)
-            .Include(i => i.Sections).ThenInclude(s => s.Items)
+            .Include(i => i.Sections).ThenInclude(s => s.Items).ThenInclude(i => i.Content).ThenInclude(c => c.FileReferences)
```

Comment:
Fix linting issues.  In this specific case you can pound it `!` because it's for LINQ to generate SQL.